### PR TITLE
zh_TW lang file

### PR DESCRIPTION
**Pull request by jh1226**

### CODE REVIEW

**Comment by Guichaguri** [comment]
Thank you, I really appreciate, but there's a problem: There are spaces before each line, can you remove them?

**Comment by Guichaguri** [comment]
Thanks :)

**Comment by jh1226** [comment]
No problem 😄

File: src/main/resources/assets/betterfps/lang/zh_tw.lang [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+
+# GUI Restart
+betterfps.options.restart.dialog.1=為應用一些修改，您需要重啟您的遊戲
+betterfps.options.restart.dialog.2=您想現在就重啟嗎？
+
+
+# GUI Algorithm Option
+betterfps.options.algorithm.title=演算法
+betterfps.options.algorithm.desc=計算正弦和余弦的演算法
```
Comment (NatsuArashi) [inline src/main/resources/assets/betterfps/lang/zh_tw.lang:78]: 餘弦?

File: src/main/resources/assets/betterfps/lang/zh_tw.lang [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+betterfps.options.allocmemory.desc=在啟動時應否預留10MB記憶體
+
+
+# GUI Fog Option
+betterfps.options.fog.title=霧
+betterfps.options.fog.desc=是否繪製霧
+
+
+# GUI Beacon Beam Option
+betterfps.options.beaconbeam.title=信標光束
```
Comment (NatsuArashi) [inline src/main/resources/assets/betterfps/lang/zh_tw.lang:110]: 好像叫做烽火台

**Pull request merged**